inc/options/class-wpseo-option-wpseo.php

```diff
@@ -256,6 +258,8 @@ protected function validate_option( $dirty, $clean, $old ) {
 				case 'semrush_country_code':
 				case 'license_server_version':
 				case 'home_url':
+				case 'category_base_url':
+				case 'tag_base_url':
```

Comment:
I'm not sure that validating `category_base_url` and `tag_base_url` this way is the best option. In WordPress, these values are passed through `sanitize_option()` in the same switch case with `permalink_structure`. Since we're using `sanitize_option()` to validate `permalink_structure`, I'd tend to think we should do the same for these two values. I'm not an expert though so please check with some senior dev or architect.

Reply:
@herregroen What do you think of this? 😄

Reply:
I agree with @afercia. If these are sanitized like that in core we should do the same to ensure the comparisons always work as expected.

This is assuming that in core the value we're reading is only sanitized we pick it up. If it's already sanitized when we get the value there's no need to sanitize it again ( in which case that may also apply to the `permalink_structure` ).